clang/include/clang/CIR/Dialect/IR/CIRAttrs.td

```diff
@@ -73,6 +73,26 @@ def LangAttr : CIR_Attr<"Lang", "lang"> {
   let genVerifyDecl = 0;
 }
 
+//===----------------------------------------------------------------------===//
+// OptInfoAttr
+//===----------------------------------------------------------------------===//
+
+def CIR_OptInfoAttr : CIR_Attr<"OptInfo", "opt_info"> {
+  let summary = "A module-level attribute that holds the optimization level";
+  let description = [{
+    The `#cir.opt_info` attribute holds the optimization related information.
+    Currently it holds the optimization level and the size optimization level.
+    This attribute is attached to the module operation during CIRGen.
```

Comment:
Please add couple simple notes on the meaning of `level` and `size`, so we can be more terse with the names while still providing extra information

Reply:
Updated.